DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -394,13 +395,48 @@ def run(self):
     part.MinDistToParentVertex= self.part.minDistToParentVertex
     part.OutputLevel = self.output.part
     part.enableUI()
+
+
+    if self.enableDetailedHitsAndParticleInfo:
```

Comment:
Move this block to the ParticleHandler helper as well.
just leave a call to the function here:
```
self.part.setDumpDetailedParticleInfo( kernel, DDG4 )
```
kernel and DDG4 should be passed as argument so we don't need to import anything in the helper, importing there would seriously slow down the response of ddsim when something goes wrong during argument parsing or when using tab completion